### PR TITLE
show the url of the error issue #922

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: services/theme_update.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def application(environ, start_response):
                 return ['']
             start_response(status, update.get_headers(len(output)))
         except:
-            log_exception(data)
```
Comment (kumar303) [inline services/theme_update.py]: `data` still looks important. Maybe turn this into `log.error('{}; data={}'.format(exc, data))` ?

File: services/utils.py [hunk 2]
```diff
@@ -145,18 +145,3 @@ def log_exception(data):
...
-         'cef.vendor': getattr(settings, 'CEF_VENDOR', 'Mozilla'),
-         'cef.version': getattr(settings, 'CEF_VERSION', '0'),
-         'cef.device_version': getattr(settings, 'CEF_DEVICE_VERSION', '0'),
-         'cef.file': getattr(settings, 'CEF_FILE', 'syslog'), }
-
-    kwargs = {'username': getattr(request, 'user', ''),
-              'signature': 'RECEIPT%s' % msg.upper(),
-              'msg': longer, 'config': c,
-              'cs2': app, 'cs2Label': 'ReceiptTransaction'}
-    return _log_cef('Receipt %s' % msg, 5, request, **kwargs)
```
Comment (kumar303) [inline services/utils.py]: looks like you can delete `_log_cef` too

**Comment by kumar303** [comment]
r+wc

File: services/theme_update.py (hunk 1 above)
Comment (andymckay) [inline services/theme_update.py]: In this case data is `data = environ['wsgi.input'].read()` which is actually an empty body. You'll see in the logs it shows `Data:  :...` and then goes on to give the traceback. So `data` is pretty damn useless at the moment.

File: services/utils.py (hunk 2 above)
Comment (andymckay) [inline services/utils.py]: will do

File: services/theme_update.py (hunk 1 above)
Comment (kumar303) [inline services/theme_update.py]: ha. I see.

**Pull request merged**